q2_fondue/citations.bib

```diff
@@ -1,10 +1,12 @@
 @article {Ziemski2022,
-	author = {Ziemski, Michal and Adamov, Anja and Kim, Lina and Fl{\"o}rl, Lena and Bokulich, Nicholas A.},
+	author = {Ziemski, Michal and Adamov, Anja and Kim, Lina and Flörl, Lena and Bokulich, Nicholas A},
 	title = {Reproducible acquisition, management, and meta-analysis of nucleotide sequence (meta)data using q2-fondue},
 	year = {2022},
-	doi = {10.1101/2022.03.22.485322},
-	URL = {https://www.biorxiv.org/content/early/2022/03/25/2022.03.22.485322},
-	journal = {bioRxiv}
+    month = {09},
```

Comment:
this line should be indented or removed. It is out of alignment with the other lines

Reply:
It seems like the indents were messed up a bit throughout the file so I just reindented everything. Hope it's fine now.